python/nammu/controller/NammuController.py

```diff
@@ -567,15 +571,20 @@ def send_request(self, client):
         try:
             client.send()
         except Timeout:
-            self.logger.error("ORACC server timed out after 5 seconds.")
-            raise
-        except ConnectionError:
-            self.logger.error("Can't connect to ORACC server at %s.",
+            self.logger.error('ORACC %s server timed out after 5 seconds.',
                               client.url)
-            raise
+            self.logger.error('You can try with a different server from the '
+                              'settings menu.')
+            raise Exception("ORACC server timed out after 5 seconds.")
```

Comment:
A minor one, but might be worth considering your own class for errors raised explicitly by Nammu.